src/Common/src/Interop/Windows/user32/Interop.EnumWindows.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class User32
+    {
+        internal delegate bool EnumThreadWindowsCallback(IntPtr hWnd, IntPtr lParam);
+
+        [DllImport(Libraries.User32)]
```

Comment:
Moving away from APISets does not mean we can use any API. Reverse Forwarders enable portable binaries so long as you consume APIs that are exposed by OneCore SDK. Thus, if this API is not in OneCore, we should not use it - otherwise, .NET Core cannot function on OneCore based OS editions.